test/manifest.ts

```diff
@@ -829,6 +850,674 @@ describe('Manifest', () => {
     });
   });
 
+  describe('githubRelease', () => {
+    it('python draft and node package', async () => {
+      const manifest = JSON.stringify({
+        'node/pkg1': '3.2.1',
+        'node/pkg2': '0.1.2',
+        python: '1.2.3',
+      });
+      const manifestConfig: ManifestConfig = {
+        packages: {
+          'node/pkg1': {},
+          python: {
+            'release-type': 'python',
+            'package-name': 'foolib',
+            'release-draft': true,
+          },
+        },
+      };
+      const config = JSON.stringify(manifestConfig);
+
+      const github = new GitHub({
+        owner: 'fake',
+        repo: 'repo',
+        defaultBranch,
+      });
+      const mock = mockGithub({
```

Comment:
@bcoe I'd already adopted this "mock all the GitHub things in one method" approach prior to your feedback in #807 about splitting it up. At this point would you be ok with a follow-on PR to refactor it (I'd refactor the existing `pullRequest` describe as well)? I'll definitely take care of it, I just have a few more changes already in the pipe that would be easier to get through and then come back to refactoring this.

Reply:
Follow on PR is absolutely fine 👍